committing.rst

```diff
@@ -460,28 +460,28 @@ Python version.
 
 .. _branch-merge:
 
-Merging between different branches (within the same major version)
-------------------------------------------------------------------
+Merging between different branches
+----------------------------------
 
 Assume that Python 3.7 is the current in-development version of Python and that
 you have a patch that should also be applied to Python 3.6.  To properly port
 the patch to both versions of Python, you should first apply the patch to
 Python 3.6::
 
-   cd 3.6
-   hg import --no-commit patch.diff
-   # Compile; run the test suite
-   hg ci -m '#12345: fix some issue.'
+   git checkout 3.6
+   git apply --reject patch.diff
+   # Fix any conflicts (e.g. look for *.rej files); compile; run the test suite
+   git add -A
+   git commit -m '#12345: fix some issue.'
+   # note the commit SHA (e.g. git log)
```

Comment:
We could also mention `git rev-parse --short HEAD` (or something else that does the same job) as an alternative here.